test/cypress/support/commands.js

```diff
@@ -29,21 +34,13 @@ Cypress.Commands.add('giLogin', (userName, password = '123456789') => {
 
   cy.getByDT('loginSubmit').click()
 
-  // Check if user as a group
-  if (cy.get('#app').find('[data-test="welcomeHomeLoggedIn"]')) {
-    cy.getByDT('welcomeHomeLoggedIn').should('contain', 'Let’s get this party started')
-  } else {
-    // For the case if the user has displayName
-    const elName = cy.getByDT('userProfile').find('[data-test="profileName"]')
-      ? 'profileName'
-      : 'profileDisplayName'
-
-    cy.getByDT(elName).should('contain', userName)
-  }
+  // We changed pages (to dashboard or create group)
+  // so there's no login button anymore
+  cy.getByDT('loginBtn').should('not.exist')
```

Comment:
NOTE: @pieer and @taoeffect There was a weird rare false positive in a test failing in this step. This was the only quick fix I found. Let me explain:

![image](https://user-images.githubusercontent.com/14869087/65779693-2798e980-e140-11e9-8564-3c88fd4ca0ae.png)

This view is visible for 1ms. It happens immediately after a user with a group logs in. How is that possible? Well...
Let's see the current conditions for each view:
- sidebar: isloggedIn and hasGroups
- welcomeGroup: isloggedIn

So, both conditions are indeed true.

If we look at the code, after the login happens, there's a tiny moment where we verify it the user has a group and then ask $router to redirect to `/dashboard`.
```js
// Login.vue#L91
await sbp('state/vuex/dispatch', 'login', { username, identityContractID })
if (this.$store.state.currentGroupId) {
  this.$router.push({ path: '/dashboard' })
}
```
That's the exactly moment where this interface (the screenshot) takes place in time.

For that reason, the condition to find "welcomeHomeLoggedIn" (L33) is positive. However, in the next millisecond, when we try to get the element again and assert the text, the element isn't there anymore because the $router has changed to `/dashboard`.

Based on this, it's a tricky task to get the correct element on the page in this moment. So I decided to change the assertions: if the login was successful, it means we won't display the login button anymore. That's it.

I think the ideal solution would be to move the welcome view (create/join group) to another router (ex: `/welcome`).  That would prevent this bug and we would still have better login assertions.

What do you think?

Reply:
> I think the ideal solution would be to move the welcome view (create/join group) to another router (ex: /welcome). That would prevent this bug and we would still have better login assertions.

In `master`, I see this this in `router.js`:

```js
    {
      path: '/welcome',
      component: GroupWelcome,
      name: GroupWelcome.name,
      meta: { title: L('Your Group Created') },
      beforeEnter: createEnterGuards(loginGuard)
    },
```

So is that not already the case?

Reply:
Hum... nop, at the moment the URL isn't `/welcome`. I believe we have a bigger issue here related to routers well. Issue #696 created